app/stories/components/ButtonLink.stories.tsx

```diff
@@ -2,9 +2,17 @@ import { ComponentMeta, ComponentStory } from '@storybook/react'
 
 import ButtonLinkComponent from '@components/ButtonLink'
 
+import ButtonStory from './Button.stories'
+
 export default {
   title: 'Components',
   component: ButtonLinkComponent,
+  argTypes: {
+    href: {
+      control: 'text',
+    },
+    style: ButtonStory.argTypes?.style,
+  },
```

Comment:
Notes for reviewers: Show the `href` arg as a text input rather than a json input. Show the button style options defined for `<Button>`